pygfx/geometries/_text.py

```diff
@@ -103,6 +111,16 @@ def ws_after(self):
         """The whitespace text after this item."""
         return self._ws_after
 
+    @property
+    def nl_before(self):
+        """The newline text in front of this item."""
+        return self._nl_before
+
+    @property
+    def nl_after(self):
+        """The newline text after this item."""
+        return self._nl_after
```

Comment:
stupid question maybe, but aren't newline also whitespace? (because I also see the properties `ws_before` and `ws_after`)

Reply:
They were actually caught by the regex handler. 

Though I decided to deal with them separately. So technically it is now “white space that is not a newline”